wrappers/aws/aws-sdk-go/aws/aws.go

```diff
@@ -13,14 +14,22 @@ import (
 )
 
 // WrapSession wraps an aws session.Session with epsgaon traces
-func WrapSession(s *session.Session) *session.Session {
+func WrapSession(s *session.Session, args ...context.Context) *session.Session {
```

Comment:
This code repeats itself for aws v1/v2 and for http. Is it possible to extract the creation of the completeEventData function with the tracer, and then each wrapper will just call that (instead of duplicating the code)?